### PR TITLE
Add support for boot-clj based projects.

### PR DESCRIPTION
**Pull request by pbzdyl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* Add [boot-clj](https://github.com/boot-clj/boot) project type.
```
Comment (bbatsov) [inline CHANGELOG.md]: You should also add a `New features` section here.

**Comment by bbatsov** [comment]
Please, remove the `.` from the commit message's title.

**Comment by pbzdyl** [comment]
@bbatsov I have added `New features` header and removed the `.` from commit message.

**Comment by bbatsov** [comment]
Don't forget to fix the changelog as well.

**Comment by pbzdyl** [comment]
Yeah, I fogot to push the change in changelog. It should be there now.

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**